go/cmd/gh-ost/main.go

```diff
@@ -149,6 +150,13 @@ func main() {
 	if migrationContext.SwitchToRowBinlogFormat && migrationContext.AssumeRBR {
 		log.Fatalf("--switch-to-rbr and --assume-rbr are mutually exclusive")
 	}
+	if *testOnReplicaWithManualReplicationControl {
+		if migrationContext.TestOnReplica {
+			log.Fatalf("--test-on-replica-manual-replication-control and --test-on-replica are mutually exclusive")
+		}
+		migrationContext.TestOnReplica = true
+		migrationContext.ManualReplicationControl = true
+	}
```

Comment:
That was my original intention. The flag is good to be named `--test-on-replica-skip-stop-replica` or whatever, but it would be an _addon_ to `--test-on-replica`. Not only are the flags not mutually exclusive, the `--test-on-replica-skip-stop-replica` is only applicable when `--test-on-replica` exists.